doc/source/rllib/getting-started.rst

```diff
@@ -0,0 +1,449 @@
+
+.. _rllib-getting-started:
+
+Getting Started
+===============
+
+.. include:: /_includes/rllib/new_api_stack.rst
+
+.. _rllib-in-60min:
+
+RLlib in 60 minutes
+-------------------
+
+.. figure:: images/rllib-index-header.svg
+
+In this tutorial, you learn how to design, customize, and run an end-to-end RLlib learning experiment
+from scratch. This includes picking and configuring an :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`,
+running a couple of training iterations, saving the state of your
+:py:class:`~ray.rllib.algorithms.algorithm.Algorithm` from time to time, running a separate
+evaluation loop, and finally utilizing one of the checkpoints to deploy your trained model
+to an environment outside of RLlib and compute actions.
+
+You also learn how to customize your :ref:`RL environment <rllib-key-concepts-environments>`
+and your :ref:`neural network model <rllib-key-concepts-rl-modules>`.
+
+Installation
+~~~~~~~~~~~~
+
+First, install RLlib, `PyTorch <https://pytorch.org>`__, and `Farama Gymnasium <https://gymnasium.farama.org>`__ as shown below:
+
+.. code-block:: bash
+
+    pip install "ray[rllib]" torch "gymnasium[atari,accept-rom-license,mujoco]"
+
+
+.. _rllib-python-api:
+
+Python API
+~~~~~~~~~~
+
+RLlib's Python API provides all the flexibility required for applying the library to any
+type of RL problem.
+
+You manage RLlib experiments through an instance of the :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`
+class. An :py:class:`~ray.rllib.algorithms.algorithm.Algorithm` typically holds a neural
+network for computing actions, called ``policy``, the :ref:`RL environment <rllib-key-concepts-environments>`
+that you want to optimize against, a loss function, an optimizer, and some code describing the
+algorithm's execution logic, like determining when to collect samples, when to update your model, etc..
+
+In :ref:`multi-agent training <rllib-multi-agent-environments-doc>`,
+:py:class:`~ray.rllib.algorithms.algorithm.Algorithm` manages the querying and optimization of multiple policies at once.
+
+Through the algorithm's interface, you can train the policy, compute actions, or store your
+algorithm's state through checkpointing.
+
+
+Configure and build the algorithm
++++++++++++++++++++++++++++++++++
+
+You first create an :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig` instance
+and change some default settings through the config object's various methods.
+
+For example, you can set the :ref:`RL environment <rllib-key-concepts-environments>`
+you want to use by calling the config's :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.environment`
+method:
+
+.. testcode::
+
+    from ray.rllib.algorithms.ppo import PPOConfig
+
+    # Create a config instance for the PPO algorithm.
+    config = (
+        PPOConfig()
+        .environment("Pendulum-v1")
+    )
+
+
+To scale your setup and define, how many EnvRunner actors you want to leverage,
```

Comment:
Shall we put all class names into ``?

Reply:
Also we might want to add that these `EnvRunner`s are used to rollout the policy and collect samples?

Reply:
done